docusaurus.config.js

```diff
@@ -26,11 +26,11 @@ module.exports = {
   themeConfig: {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     announcementBar: {
-      id: 'assembly',
+      id: 'govern',
       content:
-        'Would you like to stake your tokens? <a target="_blank" href="https://assembly.sc">Read more</a> about Assembly, the upcoming permissionless Smart Contract network.',
-      backgroundColor: '#da7ad7',
-      textColor: '#000000',
+        'If you would like to get more involved in the future Governance of Shimmer, IOTA, and the Assembly network, join the discussions with the community in our <a target="_blank" href="https://govern.iota.org">governance forum 🏛️</a>',
```

Comment:
```suggestion
        'If you would like to get more involved in the future governance of Shimmer, IOTA, and the Assembly network, join the discussions with the community in our <a target="_blank" href="https://govern.iota.org">governance forum</a> 🏛️',
```

Reply:
Not underlined icon is just my preference.
Nice color btw!

Reply:
I support this message